src/azure-cli/azure/cli/command_modules/redis/_help.py

```diff
@@ -216,3 +228,93 @@
 type: command
 short-summary: Show the identity assigned for Azure cache for Redis.
 """
+
+helps['redis access-policy'] = """
+type: group
+short-summary: Manage access policies for Redis Cache
+"""
+
+helps['redis access-policy create'] = """
+type: command
+short-summary: Add an access policy to the Redis Cache
+examples:
+    - name: Usage example
```

Comment:
`name` usually describes what this example does. eg, Add an access policy with "..." permissions

Reply:
Added a description for examples as well